linux_os/guide/services/xwindows/disabling_xwindows/xwindows_runlevel_target/bash/shared.sh

```diff
@@ -0,0 +1,7 @@
+# platform = multi_platform_sle,multi_platform_rhv,multi_platform_fedora,Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8
+# reboot = true
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+systemctl set-default multi-user.target
```

Comment:
Have you checked that `systemctl` command works during Anaconda post-install phase?

Reply:
Not yet. I'm still thinking how I can test this scenario.

Reply:
Tested RHEL7 installation providing the datastream through a webserver and customizing the kickstart to point to the custom datastream and it worked perfectly. The system ends up on a regular terminal login console

Reply:
That's great!